### PR TITLE
chore(docs): Update 21yunbox guide

### PR DESCRIPTION
**Pull request by tobyglei**
I have noticed the docs are in the reorganizing state. This PR is intended to fix the missing and/or broken links for this page: [https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/)

I have also updated the glossary to include 21YunBox.

FYI, The doc for 21YunBox deployment was merged to main doc last year from this PR: [https://github.com/gatsbyjs/gatsby/pull/24429](https://github.com/gatsbyjs/gatsby/pull/24429).

### CODE REVIEW

**Comment by LekoArts** [comment]
It seems like you didn't move the guide but created a duplicate?

**Comment by tobyglei** [comment]
> It seems like you didn't move the guide but created a duplicate?

Oh, I forgot to delete it. 

Just made the changes and committed again.

File: docs/docs/glossary/build.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Learn what _build_ means and how to set up a build process for your Gatsby proje
...
+There are a few ways to create a build. You can build your site locally on your computer using the [Gatsby CLI](/docs/reference/gatsby-cli/#build), and then deploy changes to your [host](/docs/glossary#hosting). If you use [Gatsby Cloud](https://www.gatsbyjs.com/), you can take advantage of [Gatsby Builds](/blog/2020-01-27-announcing-gatsby-builds-and-reports/), a feature available with every Gatsby Cloud account. You can also use a [continuous deployment](/docs/glossary/continuous-deployment/) service such as [AWS Amplify](/docs/how-to/previews-deploys-hosting/deploying-to-aws-amplify/), [Netlify](/docs/how-to/previews-deploys-hosting/deploying-to-netlify/), [21YunBox](/docs/how-to/previews-deploys-hosting/deploying-to-21yunbox/).
```
Comment (LekoArts) [inline docs/docs/glossary/build.md:12]: Please revert that change, it's out of scope of this PR and the paragraph is purposefully only listing those two options. Thanks!

Comment (tobyglei) [reply docs/docs/glossary/build.md]: done!

**Comment by tobyglei** [comment]
@LekoArts Hi Lennart, just a quick follow up. Is there anything I need to do to get this PR merged?

**Comment by LekoArts** [comment]
@meganesu will know :)

**Comment by meganesu** [comment]
Hi @tobyglei, thanks for your patience! As of right now, we're only moving our most frequently visited docs into the `/docs/how-to/...` folder structure.

We can merge the changes to update the links in the current deploy guide, but you'll need to move the doc back into the `/docs/docs` directory first. Thanks for your contribution!

**Comment by tobyglei** [comment]
> Hi @tobyglei, thanks for your patience! As of right now, we're only moving our most frequently visited docs into the `/docs/how-to/...` folder structure.

Just you to be aware of there are a lot of interests in deploying Gatsby in China but there is lack of supports and docs out there. I'm trying to fill in the gap here. As for the current Gatsby's docs structure it's hard to find related docs on getting Gatsby deploy in China, so that was why I moved it to the `/docs/how-to/`. 

FYI: [Why launching websites in general isn't easy in China](https://chanind.github.io/china/2019/01/19/launching-an-app-or-website-in-china.html)
 
> We can merge the changes to update the links in the current deploy guide, but you'll need to move the doc back into the `/docs/docs` directory first. Thanks for your contribution!

Will do.

**Review by LekoArts — APPROVED** [review]

**Pull request merged**